app/controllers/posts_controller.rb

```diff
@@ -25,6 +25,17 @@ def destroy
     redirect_to '/posts/'
   end
 
+  def toggle_like_post
+    @post = Post.find(params[:post_id])
+    @user = current_user
```

Comment:
keep this consistent between the two controllers? either @user = current_user or use current_user directly